src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -680,9 +713,62 @@ protected virtual ClaimsIdentity CreateClaimsIdentity(JsonWebToken jwtToken, Tok
             if (string.IsNullOrWhiteSpace(issuer))
                 issuer = GetActualIssuer(jwtToken);
 
+            if (DefaultMapInboundClaims)
```

Comment:
this branch looks like it is missing new test coverage?

Reply:
thanks for catching that. The theory data wasn't working how I thought, so there's a new test.